hhvm/content.md

```diff
@@ -0,0 +1,73 @@
+# What is HHVM?
+
+HHVM is an open-source virtual machine designed for executing programs written in Hack and PHP. HHVM uses a just-in-time (JIT) compilation approach to achieve superior performance while maintaining the development flexibility that PHP provides.
+
+> [wikipedia.org/wiki/HipHop_Virtual_Machine](https://en.wikipedia.org/wiki/HipHop_Virtual_Machine)
+
+%%LOGO%%
```

Comment:
The change here looks good. I think you may want to regenerate the `README.md` though. I'm not sure whether `README.md` should actually be left out of initial doc submission and allowed to be generated after the PR merges, though.